pkg/internal/checkup/executor/trex-console.go

```diff
@@ -34,8 +35,10 @@ func (t trexConsole) ClearStats(ctx context.Context) (string, error) {
 	return t.runCommand(ctx, "clear")
 }
 
-func (t trexConsole) StartTraffic(ctx context.Context, packetPerSecondMillion, port int) (string, error) {
-	return t.runCommand(ctx, fmt.Sprintf("start -f /opt/tests/testpmd.py -m %dmpps -p %d", packetPerSecondMillion, port))
+func (t trexConsole) StartTraffic(ctx context.Context, packetPerSecondMillion, port int, testDuration time.Duration) (string, error) {
+	testDurationMinutes := int(testDuration.Minutes())
+	return t.runCommand(ctx, fmt.Sprintf("start -f /opt/tests/testpmd.py -m %dmpps -p %d -d %dm",
+		packetPerSecondMillion, port, testDurationMinutes))
```

Comment:
Please consider building this command as a separate initial step.
IMO it will improve readability.

Reply:
I plan to do it a a future PR.

#2DO RAM